pytorch_lightning/callbacks/stochastic_weight_avg.py

```diff
@@ -115,6 +115,7 @@ def __init__(
         if device is not None and not isinstance(device, (torch.device, str)):
             raise MisconfigurationException(f"device is expected to be a torch.device or a str. Found {device}")
 
+        self.n_averaged = None
```

Comment:
```suggestion
        self.n_averaged: Optional[torch.Tensor] = None
```

Reply:
Fixed, plus I added annotations to some of the other members below too